label_candidates/label_candidates.py

```diff
@@ -0,0 +1,254 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:snorkeling]
+#     language: python
+#     name: conda-env-snorkeling-py
+# ---
+
+# # Label the Candidate Sentences
+
+# +
+# %load_ext autoreload
+# %autoreload 2
+
+import os
+import sys
+
+from sqlalchemy import create_engine
+from snorkel.labeling import PandasLFApplier
+import pandas as pd
+
+from snorkeling_helper.label_functions.disease_gene import DaG
+# -
+
+username = "danich1"
+password = "snorkel"
+dbname = "pubmed_central_db"
+database_str = (
+    f"postgresql+psycopg2://{username}:{password}@/{dbname}?host=/var/run/postgresql"
+)
+conn = create_engine(database_str)
+
+candidates_df = pd.read_csv(
+    "../annotation_conversion/output/mapped_disease_gene_candidates.tsv", sep="\t"
+)
+candidates_df.head()
+
+# ## Train Candidates Abstract Only
+
+# +
+candidates = ",".join(map(str, candidates_df.document_id.tolist()))
+sql = f"""
+select * from disease_gene
+where sentence_id not in ({candidates}) and
+section = 'title' or section ='abstract'
+"""
+
+candidate = pd.read_sql(sql, database_str)
+print(candidate.shape)
+candidate.head()
+
+# +
+sen_id_map = ",".join(map(str, candidate.sentence_id.tolist()))
```

Comment:
Nice clean code. realize much of this is redundant to the old repo, but if people are going to be looking at this one in the future, might be helpful to add more comments about what this notebook is doing..Or maybe you plan to add this in the readme later

Reply:
I agree it would be easier to add description into this notebook, so I'll add that. Good catch!